src/api/users.js

```diff
@@ -0,0 +1,10 @@
+const fetchUsers = async () => {
+  // TODO: Update to use vdir backend node app
+  const response = await fetch("https://randomuser.me/api/?results=10");
+  const data = await response.json();
+  return data;
+};
+
+module.exports = {
+  fetchUsers,
+};
```

Comment:
this _should_ have a suggestion that it can be converted into an ES module - we should follow the convention here too I think?

```suggestion
export default fetchUsers // if you want it default
```

```suggestion
export const fetchUsers... // if you want to match your current
```

Reply:
Updated: [20f1fbd](https://github.com/8rianHo/vietnamese-directory/pull/2/commits/20f1fbdf56d23a28c314773f7a0974040eadce8c)